lib/presentation/feature/news/widgets/top_anime_item_widget.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../models/anime_news_item_model.dart';
+
+class TopAnimeItemWidget extends ConsumerWidget {
+  const TopAnimeItemWidget({
+    required this.animeNewsItemModel,
+    Key? key,
+  }) : super(key: key);
+  final AnimeNewsItemModel animeNewsItemModel;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return GridTile(
+      footer: Material(
+        color: Colors.transparent,
+        shape: const RoundedRectangleBorder(
+          borderRadius: BorderRadius.vertical(bottom: Radius.circular(4)),
+        ),
+        clipBehavior: Clip.antiAlias,
+        child: GridTileBar(
+          backgroundColor: Colors.black45,
+          title: _GridTitleText(animeNewsItemModel.titleEn),
+          subtitle: _GridTitleText(animeNewsItemModel.titleJp),
+        ),
+      ),
+      child: Material(
+        color: Colors.black12,
+        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(4)),
+        clipBehavior: Clip.antiAlias,
+        child: FadeInImage.assetNetwork(
+          placeholder: 'assets/images/loading.gif',
```

Comment:
Same for the images app_images, we can use the same files as the current flutter_template and reuse the same structure and name files

Reply:
Done! Please see the newly created `images.dart`.